File: 26_RemoveDuplicatesFromSortedArray/c#/RemoveDuplicatesFromSortedArray.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if(nums.Length<2){
            return nums.Length;
        }
        int len = 0;
        for(int i = 1; i<nums.Length;i++){
            if(nums[len]!=nums[i]){
                len++;
                nums[len]=nums[i];
            }
        }
        
        len++;
        return len;
    }
}

File: 26_RemoveDuplicatesFromSortedArray/python/RemoveDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        length = 0
        
        for idx in range(1, len(nums)):
            if nums[length] != nums[idx]:
                length += 1
                nums[length] = nums[idx]
                
        return length + 1